docs/bnb-smart-chain/governance/temp-check.md

```diff
@@ -0,0 +1,27 @@
+---
+title:  Temperature Check - BSC Governance
+---
+# BSC Governance Temperature Check
+
+
+## Overview
+
+BNB Chain governance involves a two-step process: temperature check and final decision voting. The temperature check, typically conducted through the Snapshot platform, allows any BNB holder to gauge community sentiment on a proposal. If the proposal receives enough support, it proceeds to the final decision voting phase. This phase often involves on-chain voting by validators or those with staked BNB, and the outcome determines whether the proposal is implemented or rejected.
+
```

Comment:
Can you also add the `submit proposal ` section?

And also hightlight who can vote?